site/src/pages/ManagementSettingsPage/IdpSyncPage/IdpSyncPageView.tsx

```diff
@@ -63,7 +65,7 @@ export const IdpSyncPageView: FC<IdpSyncPageViewProps> = ({
 	return (
 		<>
 			<ChooseOne>
-				<Cond condition={false}>
+				<Cond condition={!isIdpSyncEnabled}>
 					<Paywall
 						message="IdP Sync"
 						description="Configure group and role mappings to manage permissions outside of Coder. You need an Premium license to use this feature."
```

Comment:
Is this rendering properly? The paywall in the screenshot has alignment values all over the place

Reply:
yes, there were previously 2 changes to increase the text length of the right side bullet points and make the banner overall wider. Can you create a new issue for the improvements you would like to see?